store/tikv/lock_resolver.go

```diff
@@ -52,37 +49,6 @@ func newLockResolver(store Storage) *LockResolver {
 	return r
 }
 
-// NewLockResolver creates a LockResolver.
-// It is exported for other services to use. For instance, binlog service needs
-// to determine a transaction's commit state.
-func NewLockResolver(etcdAddrs []string, security config.Security) (*LockResolver, error) {
```

Comment:
This is removed?

Reply:
yes~I found the method is unused code

Reply:
Binlog use it!

Reply:
Maybe you could update the comment, add "DO NOT REMOVE IT"

Reply:
done